rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -2549,6 +2549,10 @@ void settings_dialog::AddStylesheets()
 {
 	ui->combo_stylesheets->clear();
 
+#ifdef __APPLE__
+	//Default style needs to be native on macOS for readability when system Dark Mode is enabled.
```

Comment:
We're not supporting backwards compatibility unless it's really necessary.
Which is hardly the case here

Reply:
This was meant for the other comment 😂

Reply:
We're not restricting users just because one person thinks something is not necessary. If that was the case, I wouldn't have done 90% of the user interface.
Also, none and native is NOT the same